client/src/utils/routes.js

```diff
@@ -0,0 +1,62 @@
+export const ADMIN = 'ADMIN';
+export const ANY = 'ANY';
+export const HOME = 'Home';
+export const SQUAD_LEADER = 'SQUAD_LEADER';
+
+export const canUserSeeLink = (link, user = {}) =>
```

Comment:
Also, in `auth.js`. It has a function called `userCanAccess`. This one is called `canUserSeeLink`. This bothers me. Fix it, self